metabase/metabase
# Make the Between filter easier to use and understand

Epic https://github.com/metabase/metabase/issues/25026
Demo 1 https://www.loom.com/share/c6e28955f0e848c097242009af55ef18
Demo 2 https://www.loom.com/share/8fcf1d1da8144545b4bc6c3ded7c3695

How to test:
- Open Orders table
- Click on `Created At` -> `Filter by this column` -> `Specific Dates`
- It should open `Between` tab with the date range from 30 days ago through today
- Make sure it is possible to change the date range by clicking on dates in the calendar and by typing in inputs
